src/jit/gentree.h

```diff
@@ -1628,6 +1628,9 @@ struct GenTree
 #ifdef FEATURE_SIMD
             case GT_SIMD:
 #endif // !FEATURE_SIMD
+#ifdef _TARGET_ARM_
+            case GT_PUTARG_REG:
+#endif
                 return true;
```

Comment:
I have changed `GT_PUTARG_REG` to be `GenTreeMultiRegOp`(a subclass of `GenTreeOp`) for ARM. That's why it was necessary.